.github/workflows/build.yml

```diff
@@ -40,13 +72,23 @@ jobs:
         run: |
           docker login --help
           docker login -u $USERNAME -p $PASSWORD quay.io
-          docker tag quay.io/hawtio/hawtio-online-example-camel-quarkus:latest quay.io/hawtio/hawtio-online-example-camel-quarkus:${{matrix.java}}
-          docker tag quay.io/hawtio/hawtio-online-example-camel-springboot:latest quay.io/hawtio/hawtio-online-example-camel-springboot:${{matrix.java}}
 
-          docker push quay.io/hawtio/hawtio-online-example-camel-quarkus:${{matrix.java}}
-          docker push quay.io/hawtio/hawtio-online-example-camel-springboot:${{matrix.java}}
+          mvn --batch-mode --no-transfer-progress -pl '!camel-quarkus-openshift' install \
+            -Dquarkus.jib.base-jvm-image=registry.access.redhat.com/ubi8/openjdk-${{ matrix.java }}-runtime:latest \
+            -Dquarkus.jib.platforms=linux/amd64,linux/arm64,linux/ppc64le,linux/s390x \
+            -Dquarkus.container-image.name=hawtio-online-example-camel-quarkus \
+            -Dquarkus.container-image.tag=${{ matrix.java }} \
+            -Dquarkus.container-image.push=true \
+            -Djkube.generator.from=quay.io/hawtio/jkube-java-${{ matrix.java }}:latest \
+            -Djkube.build.strategy=docker
+
+          docker buildx build -t quay.io/hawtio/hawtio-online-example-camel-springboot:${{ matrix.java }} \
+            --platform linux/amd64,linux/arm64,linux/ppc64le,linux/s390x \
+            --provenance=false \
+            --push \
+            camel-springboot/target/docker/quay.io/hawtio/hawtio-online-example-camel-springboot/latest/build/
 
-          if [[ "${{matrix.java}}" == "21" ]]; then
-            docker push quay.io/hawtio/hawtio-online-example-camel-quarkus:latest
-            docker push quay.io/hawtio/hawtio-online-example-camel-springboot:latest
+          if [[ "${{ matrix.java }}" == "21" ]]; then
+            docker buildx imagetools create -t quay.io/hawtio/hawtio-online-example-camel-quarkus:latest quay.io/hawtio/hawtio-online-example-camel-quarkus:${{ matrix.java }}
+            docker buildx imagetools create -t quay.io/hawtio/hawtio-online-example-camel-springboot:latest quay.io/hawtio/hawtio-online-example-camel-springboot:${{ matrix.java }}
```

Comment:
I'm assuming this command also pushes right?

Reply:
Yes, this command retags the 2 images to latest. Since the image that is being retagged is multi-arch, it is done this way using buildx.